virtool/auth/utils.py

```diff
@@ -90,4 +113,15 @@ async def write_auth_model(api_instance):
         )
 
         await api_instance.write_authorization_model(type_definitions)
-        logger.info("Writing authorization model")
+
+
+async def check_openfga_version(client: ApiClient):
+    """
+    Check the OpenFGA version.
+
+    :param client: the application client object
```

Comment:
Not specific enough. Please correct this elsewhere as well.
```suggestion
    :param client: the application OpenFGA client
```